WatchApp Extension/Scenes/GlucoseChartScene.swift

```diff
@@ -235,10 +238,12 @@ class GlucoseChartScene: SKScene {
         addChild(now)
 
         hoursLabel = SKLabelNode.basic(at: CGPoint(x: textInsets.left, y: size.height - textInsets.top))
+        hoursLabel.verticalAlignmentMode = .top
```

Comment:
This is already set to .top in the basic() initializer above.

Reply:
Hmm, I see that now.  That label was failing to align correctly when I was trying alternate scene sizes, and this fixed it.  I'll see if I can reproduce that later, but probably can revert this.

Reply:
Can't reproduce what I saw, so I've reverted those changes.